### DISCUSSION TITLE
Klipperscreen reports as working but no display is being shown

### DISCUSSION THREAD

**Discussion by Rubiniuz** [KlipperScreen/KlipperScreen#1048] [Q&A]
I have a Odroid N2+ running armbian [https://www.armbian.com/odroid-n2/](url) because of driver support

I haven't had issues before installing klipperscreen. I got a terminal and resolution and refresh rates where all to spec of my display.
Edit: Now it runs the terminial until you see starting klipperscreen. when klipperscreen starts I get a no signal popup from my screen.
I have a brandless 7 inch portable display [https://www.aliexpress.com/item/1005004163930837.html?spm=a2g0o.order_list.order_list_main.17.69b81802YBH8Yv](url). 
It works perfect on ubuntu and on windows with the basic drivers.

running systemctl status Klipperscreen.service gives me a running checkmark and 0 errors or warnings.
I haven't changed the default configs of the os.

I am really stuck and can't find anything. Any ideas are appreciated.

**Comment by alfrix** [comment]
need log files to start troubleshooting https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/

**Reply by Rubiniuz** [reply]
So thats the thing. When i uninstalled klipperscreen it also deleted the log files. After installing it and running it. it hasnt generated new log files yet. But ill be able to check it again next day
